frontend/app/work_packages/services/work-package-field-service.js

```diff
@@ -53,6 +53,9 @@ module.exports = function(
     // TODO: extract to strategy if new cases arise
     if (field === 'date') {
       // nope
+      if(workPackage.links.children) {
+        return false;
+      }
```

Comment:
That is why I talked to you in gitter:
> @hirtie-maxim Are you respecting the writable-property of the WP form? (asking because of https://community.openproject.org/work_packages/19903)
> 
> What I want to say is: Don't check whether it is a parent element or anything. Check whether it is writable according to the WP form